Jenkinsfile.deploy

```
@@ -69,9 +69,51 @@ pipeline {
                         echo "Set Maven mirror url"
                         env.MAVEN_MIRROR_URL = env.MAVEN_MIRROR_REPOSITORY
                     }
+                    if (params.RELEASE) {
+                        assert params.PROJECT_VERSION != ''
+                    }
+                    setDeployPropertyIfneeded('git.branch', params.BUILD_BRANCH_NAME)
+                    setDeployPropertyIfneeded('project.version', params.PROJECT_VERSION)
+                    setDeployPropertyIfneeded('release', params.RELEASE)
+
+                }
+            }
+        }
+    
+
+        stage('Prepare for PR') {
+            when {
+                expression {
+                    return params.RELEASE
+                }
+            }
+            steps {
+                script {
+                    githubscm.forkRepo( "bsig-gh-bot" )
+                    githubscm.createBranch(params.VERSION)
```

Leftover from renaming :)

```suggestion
                    githubscm.createBranch(params.PROJECT_VERSION)
```